src/application/clients/crux-wallet-client.ts

```diff
@@ -174,15 +189,27 @@ export class CruxWalletClient {
         return {success, failures};
     }
 
+    @throwCruxClientError
+    public putParentAssetFallbacks = async (symbolAssetGroups: string[]): Promise<string[]> => {
```

Comment:
1. Can you add some inline documentation here
2. Naming consistency with `symbolAssetGroups`